docker/utils.js

```diff
@@ -8,7 +8,7 @@ const { htmlHintConfig, fetchHtml, getHTMLHintRules } = require("./api");
 const R = require("ramda");
```

Comment:
why is this changed? I don't see any changes apart from formatting changes

Reply:
Yes - they are only formatting changes